easybuild/easyconfigs/r/RNAz/RNAz-2.1-foss-2016b.eb

```diff
@@ -0,0 +1,25 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+
+easyblock = 'ConfigureMake'
+
+name = 'RNAz'
+version = '2.1'
+
+homepage = 'http://www.tbi.univie.ac.at/~wash/RNAz/'
+description = """RNAz is a program for predicting structurally conserved and
+thermodynamically stable RNA secondary structures in multiple sequence alignments."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'optarch': True, 'pic': True}
```

Comment:
@SethosII `optarch` is enabled by default, so you can remove it from `toolchainopts`

to control the C standard, you can use `'cstd': 'gnu89'`, so:

```python
toolchainopts = {'cstd': 'gnu89', 'pic': True}
```